ImageNet/train/diux_resnet152_p3-tensorpack.json

```diff
@@ -0,0 +1,24 @@
+{
+	"version": "v1.0",
+	"author": "DIUX",
+	"authorEmail": "ybulatov.ctr@diux.mil",
+	"framework": "tensorflow 1.5, tensorpack 0.8.1",
+	"codeURL": "https://github.com/diux-dev/cluster/blob/master/dawn/resnet.b512.baseline.py",
+	"model": "ResNet152",
+	"hardware": "p3.16xlarge",
+	"timestamp": "2018-01-31",
+  "costPerHour": 24.48,
+  "optimizer": "SGD with Momentum",
+  "momentum": 0.9,
+  "schedule": [
+    {"learning_rate": 0.1, "epochs": 1},
+    {"learning_rate": 0.15, "epochs": 1},
+    {"learning_rate": 0.2, "epochs": 28},
+    {"learning_rate": 0.02, "epochs": 30},
+    {"learning_rate": 0.002, "epochs": 20},
+    {"learning_rate": 0.0002, "epochs": 20}
+  ],
+	"misc": {
+		"ami": "ami-9ba7c4e1 (Amazon Deep Learning AMI version 2, Ubuntu, conda based)"
```

Comment:
You used "AMI" for the CIFAR10 experiments. It would be nice if they were consistent.